building_server/server.py

```diff
@@ -10,7 +10,10 @@
 import json
 import numpy as np
 
+# This file holds the functions of the rest API
 
+# Returns the geometry of a tile
+# args : city ; tile ; format ; attributes (what is that ?)
```

Comment:
> (what is that ?)

I think the attribute retrieval is broken at the moment, it has not yet been adapted to 3d citydb.